ivy/functional/frontends/tensorflow/math.py

```diff
@@ -226,6 +227,15 @@ def cumsum(x, axis, exclusive=False, reverse=False, name=None):
     )
 
 
+@to_ivy_arrays_and_back
+def digamma(x, name=None):
+    backend = _get_backend(x)
+    if backend == "tensorflow":
+        return ivy.array(ivy.digamma(x), ivy.array([0], ivy.int32))
+    else:
+        raise NotImplementedError(f"digamma is not implemented for backend: {backend}")
```

Comment:
Please don't do this. If the `ivy.digamma` function only does the right calculation with the tensorflow backend, please raise an issue to let us know - it should be the same regardless of backend.

Also, could you explain the composition you have here? Why not simply `return ivy.digamma(x)`?